### PR TITLE
[RUBY-2667] transfer link should be disabled once frontend users are removed

### PR DESCRIPTION
**Pull request by brujeo**
https://eaflood.atlassian.net/browse/RUBY-2667

### CODE REVIEW

File: app/helpers/action_links_helper.rb [hunk 1]
```diff
@@ -153,11 +153,8 @@ def display_renew_link_for?(resource)
     resource.can_start_renewal?
   end
 
-  def display_transfer_link_for?(resource)
-    return false if WasteCarriersEngine::FeatureToggle.active?(:block_front_end_logins)
-    return false unless display_registration_links?(resource)
-
-    can?(:transfer_registration, WasteCarriersEngine::Registration)
+  def display_transfer_link_for?(_resource)
```
Comment (PaulDoyle-DEFRA) [inline app/helpers/action_links_helper.rb:156]: I think this should just be removed

Comment (brujeo) [reply app/helpers/action_links_helper.rb:156]: removed

Comment (PaulDoyle-DEFRA) [reply app/helpers/action_links_helper.rb:156]: It's still there

Comment (brujeo) [reply app/helpers/action_links_helper.rb]: looks like my previous commit hasn't been pushed. That code is gone now

**Review by PaulDoyle-DEFRA — APPROVED** [review]

**Pull request merged**